src/GraphQL.ApiTests/GraphQL.approved.txt

```diff
@@ -1263,7 +1263,6 @@ namespace GraphQL.Language.AST
         public Directives() { }
         public override System.Collections.Generic.IEnumerable<GraphQL.Language.AST.INode> Children { get; }
         public int Count { get; }
-        public bool HasDuplicates { get; }
```

Comment:
Let's note the API changes in the migration document, or at least mention them.  Is this a new class?  It would not need to be mentioned then of course..